packages/medusa/src/services/order-edit.ts

```diff
@@ -389,6 +392,67 @@ export default class OrderEditService extends TransactionBaseService {
     })
   }
 
+  async removeLineItem(
+    orderEditId: string,
+    lineItemId: string
+  ): Promise<OrderEdit> {
```

Comment:
suggestion: Not sure we should return the order edit, a bit like the update since it would only return a partial order edit with some fields and the consumer will probably re fetch it with the relations and fields that he needs. Like we do in the handler. wdyt?